cluster/stone-prd-rh01/tenants/konflux-rh-platform-experience-tenant/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+resources:
+  - user-preferences-frontend_integration_test.yaml # has to match the name of the file created in the next step
+
+  apiVersion: kustomize.config.k8s.io/v1beta1
+  kind: Kustomization
```

Comment:
These two lines are badly indented. They need to be at the same level of `resources`